cmd/vulcan-cli/cmd/users.go

```diff
@@ -0,0 +1,81 @@
+/*
+Copyright 2023 Adevinta
+*/
+
+package cmd
+
+import (
+	"fmt"
+	"os"
+	"slices"
+
+	"github.com/adevinta/vulcan-api/cmd/vulcan-cli/cli"
+	"github.com/spf13/cobra"
+)
+
+var users = &cobra.Command{
+	Use:   "users <output_file>",
+	Short: "Writes the list of the emails corresponding to all the users and recipients in Vulcan to the specified output file.",
+	Args:  cobra.ExactArgs(1),
+	RunE: func(cmd *cobra.Command, args []string) error {
+		return runUsers(args[0], apiClient)
+	},
+}
+
+func init() {
+	users.Flags().BoolVarP(&force, "force", "f", false, "overwrites the output file if it exits")
+	rootCmd.AddCommand(users)
+}
+
+func runUsers(outputFile string, apiCLI *cli.CLI) error {
+	// Check for the existence of the file before performing the calls to
+	// the vulcan api to avoid the user to wait just to see the command fail.
+	exists, err := fileExists(outputFile)
+	if err != nil {
+		return err
+	}
+	if exists && !force {
+		return ErrOutputAlreadyExists
+	}
```

Comment:
```suggestion
	if !force {
		// Check for the existence of the file before performing the calls to
		// the vulcan api to avoid the user to wait just to see the command fail.
		exists, err := fileExists(outputFile)
		if err != nil {
			return err
		}
		if exists {
			return ErrOutputAlreadyExists
		}
	}
```

I think this is a bit more readable.